sdks/smart-wallet-sdk/abis/MinimalDelegationEntry.json

```diff
@@ -0,0 +1 @@
e207d9617b0db38679ec955dc9ec6a4f4e4a81b7672e4","dweb:/ipfs/QmYqMoV7BwzeqM6KY8kFBraPf2hanj2xLd6S878yAxuvrh"],"license":"UNLICENSED"},"src/libraries/CalldataDecoder.sol":{"keccak256":"0x189f333f20d5727ad08f0b21b43169c2f08abe757a6dd78e34f6f987db04bbef","urls":["bzz-raw://01b4eadd667b47418c10d66b0a56070f8df4be05e8b6e885748275bd5b517c93","dweb:/ipfs/QmZGQYqvjJGiUzxaimbNPJ465E2Na7CG3v39etdjRpmYFq"],"license":"UNLICENSED"},"src/libraries/EntrypointLib.sol":{"keccak256":"0xb8a68c7c3f7fa415691d864b632ea6386c3595501ff9ce559dee189c0f30b9c9","urls":["bzz-raw://19dd6eeacf1c768b749a9f1c79177ff35937247782fa1ab10edee8e6fb5c1695","dweb:/ipfs/QmecuMg3x8K6T8YzbPWwdJDVsdhnGdM4xALR3DHy3VRuc2"],"license":"UNLICENSED"},"src/libraries/HooksLib.sol":{"keccak256":"0xa9273a835bdcabb06f769fc228c4f357691c492d67c8f20fe6533fe75b09f491","urls":["bzz-raw://fff395423793a97f9016cae75da4eaad773ac22f5a05bb6fffe69a073ccedf20","dweb:/ipfs/QmZfwTATGE6KEwdg1m5QXJv33xPTCSZyG77hBswev8JMfY"],"license":"UNLICENSED"},"src/libraries/KeyLib.sol":{"keccak256":"0x2b8303c14ec6c2b7d53c50ce13b3b965912eb235a141a91ea5469bafc10fa8e8","urls":["bzz-raw://e45f0877b9994064029eec77cc7966d646b90a97c1047df6e4ac203291f85214","dweb:/ipfs/QmeFCLY5ucb9KfroaoCZcFsUXc85cdge62pTP1v35qXxcx"],"license":"UNLICENSED"},"src/libraries/ModeDecoder.sol":{"keccak256":"0xdb6231fe33c822bc9967334e4ae7b9ca608cab46b56a2a5ef91b81c5dd91d665","urls":["bzz-raw://cb551ee480ef8a9afae5f700edc25e1cc0c2040f0a3dcc1dfca14511b4b6efea","dweb:/ipfs/QmXByzVBH4BRZj1ME9KYihJbrSHanJRAp6WeznS3i2cV1t"],"license":"UNLICENSED"},"src/libraries/SettingsLib.sol":{"keccak256":"0xb4bcc64fb4d508533e5df737c0ae05bfba1a72537e7e64af8096ffc6dbabfabc","urls":["bzz-raw://305c9340c5290a83ba7eb2df2ca35edc6ad6561b817426cf0fb9daab9ea2189c","dweb:/ipfs/QmQtb4KmhpRYBVsqjv6YWtV39bYdsrGp5w3L24vo1oGLgR"],"license":"UNLICENSED"},"src/libraries/SignedBatchedCallLib.sol":{"keccak256":"0x5b46f93d6715cc29bdd5591eadbf7240390621cc89f183d16b550c2754487d3c","urls":["bzz-raw://f5cb725f79e019d6610ae3b73d50b8af423b631832be9f0592ae9df4e7a91230","dweb:/ipfs/QmQE7wM39w9CxDHk3xKoJ7YRwNEZUaQPHM2Fz2EZss3Dmp"],"license":"MIT"},"src/libraries/Static.sol":{"keccak256":"0xaf78f335887768df4267e8abf6f4acbd423c07f69d576bed84070cbcca35480c","urls":["bzz-raw://2d6c44dd8671b62275225ad520d8627af8928f03733e5e76e9f0e53b6cf3c76e","dweb:/ipfs/QmWGKvtYx9jC7RfdZCG6Avy9ZKPfALrYpSMFNGSY13uHPX"],"license":"UNLICENSED"},"src/libraries/TransientAllowance.sol":{"keccak256":"0x2dba244824723e9538c0c280726aad53a79aca876b589a04c368b31acab653cc","urls":["bzz-raw://6039868a862134a6e2a641be75b46a7759d5b867e6608c91545c6c770f60102e","dweb:/ipfs/QmRosmREFFhCNEV5WaNmFYgLd9Qq1qhSsdtwgy9uFpCMPs"],"license":"UNLICENSED"},"src/libraries/WrappedDataHash.sol":{"keccak256":"0x81d94e804efa4778c1f7098f9e7d9fda46b0fbc5fb149efcb323768fe3e4d4a4","urls":["bzz-raw://d950ad8309c651efe4f291db68343346b92a45d3198791ab6610bb3937226440","dweb:/ipfs/Qmch5Jk29XUivf5tJs2RboPMCU3pZqbrY2Vb5F6YEbJ4Cb"],"license":"UNLICENSED"}},"version":1},"id":41}
```

Comment:
run 

`forge inspect MinimalDelegationEntry abi --json > MinimalDelegationEntry.json` and you'll just get the abi